### PR TITLE
PYTHON-3567 Add guidance for setting uuidRepresentation in 4.0 migration guide

### PR DESCRIPTION
**Pull request by juliusgeo**

### CODE REVIEW

File: doc/migrate-to-pymongo4.rst [hunk 1]
```diff
@@ -980,6 +980,13 @@ The default uuid_representation for :class:`~bson.codec_options.CodecOptions`,
 :data:`bson.binary.UuidRepresentation.PYTHON_LEGACY` to
 :data:`bson.binary.UuidRepresentation.UNSPECIFIED`. Attempting to encode a
 :class:`uuid.UUID` instance to BSON or JSON now produces an error by default.
+If you were using UUIDs previously, you will need to set your uuid_representation to
```
Comment (blink1073) [inline doc/migrate-to-pymongo4.rst:983]: uuid_representation should be in double-backticks.

**Review by blink1073 — CHANGES_REQUESTED** [review]

File: doc/migrate-to-pymongo4.rst (hunk 1 above)
Comment (blink1073) [reply doc/migrate-to-pymongo4.rst:983]: Yes, I think so

Comment (juliusgeo) [reply doc/migrate-to-pymongo4.rst]: Done.

File: doc/migrate-to-pymongo4.rst [hunk 2]
```diff
@@ -974,12 +974,19 @@ subdocument containing a ``$ref`` field would be decoded as a
...
 :data:`bson.binary.UuidRepresentation.PYTHON_LEGACY` to
 :data:`bson.binary.UuidRepresentation.UNSPECIFIED`. Attempting to encode a
 :class:`uuid.UUID` instance to BSON or JSON now produces an error by default.
+If you were using UUIDs previously, you will need to set your ``uuid_representation`` to
+:data:`bson.binary.UuidRepresentation.PYTHON_LEGACY` to avoid data corruption. If you do not have UUIDs,
+then you should set :data:`bson.binary.UuidRepresentation.STANDARD`. If you do not explicitly set a value,
+you will receive an error like this when attempting to encode a :class:`uuid.UUID`::
+
+    ValueError: cannot encode native uuid.UUID with UuidRepresentation.UNSPECIFIED. UUIDs can be manually converted...
+
```
Comment (ShaneHarvey) [inline doc/migrate-to-pymongo4.rst:989]: We should reiterate that uuid representation can be set on a client here, like this:
```
client = MongoClient(uri, uuidRepresentation="pythonLegacy")
client.test.test.insert_one({'uuid': uuid.uuid4()})
```

**Pull request merged**